sections/earn/IncentivesTable.tsx

```diff
@@ -187,7 +187,7 @@ const IncentivesTable: FC<IncentivesTableProps> = ({ data, isLoaded, activeTab }
 				Header: <>{t('earn.incentives.options.rewards.title')}</>,
 				accessor: 'rewards',
 				Cell: (cellProps: CellProps<EarnItem, EarnItem['rewards']>) => {
-					const isDualRewards = cellProps.row.original.staked.asset === LP.UNISWAP_DHT;
+					const isDualRewards = typeof cellProps.row.original.rewards === 'object';
```

Comment:
this doesn't feel like a good solution. can we do better?

Reply:
yeah good point, fixed with something better